src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -68,9 +68,14 @@ struct RpmOstreeSysrootUpgrader {
 
   OstreeDeployment *cfg_merge_deployment;
   OstreeDeployment *origin_merge_deployment;
-  RpmOstreeOrigin *original_origin;
   RpmOstreeOrigin *origin;
+
+#if 0
+  /* XXX: if this is ever re-enabled, it should be part of
+   * the origin so that it can be made persistent */
   GHashTable *ignore_scripts;
+#endif
```

Comment:
My vote is to delete this code rather than `#if 0` - can you do that as a separate patch?